src/main.py

```diff
@@ -263,6 +263,9 @@ def __init__(self, aconfig):
 
 		self._aconfig = aconfig
 
+		if not self._aconfig['refresh_token']:
```

Comment:
Ah, sorry, that token is something different to what I thougt. I think that it is better to change message describing what we were after and what needs to be added to config or what to run to re-add it (I guess this is web_auth?)

Having messages that describe possible fix will reduce support load on us in future if users can find and fix errors themselves.